www-apps/gitea/gitea-1.9.0.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit golang-vcs-snapshot tmpfiles systemd user
+
+EGO_PN="code.gitea.io/gitea"
+
+DESCRIPTION="A painless self-hosted Git service"
+HOMEPAGE="https://gitea.io"
+SRC_URI="https://github.com/go-gitea/gitea/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64"
+IUSE="pam sqlite"
+
+COMMON_DEPEND="pam? ( sys-libs/pam )"
+DEPEND="${COMMON_DEPEND}
+	dev-go/go-bindata"
+RDEPEND="${COMMON_DEPEND}
+	dev-vcs/git"
+
+DOCS=( custom/conf/app.ini.sample CONTRIBUTING.md README.md )
+S="${WORKDIR}/${P}/src/${EGO_PN}"
+
+PATCHES=( "${FILESDIR}/gitea-mod-vendor.patch" "${FILESDIR}/gitea-logflags.patch" )
+
+pkg_setup() {
+	enewgroup git
```

Comment:
Please use the new GLEP-81 way of handling user and group creation, or dependency. I'm actually amazed git hasn't been added to acct-*/ yet.

Let me know if you need help with it, although if you follow gentoo-dev, it should be pretty clear what to do next.

Reply:
the main problem I have is the home dir - how should that be handled regarding other packages like gitolite?

Reply:
ACCT_USER_HOME can be specified in the virtual ebuild using acct-user.eclass. Or are you asking how should gitolite be updated? I think it just needs to RDEPEND on these acct-*/ virtuals and keep using fowners git:git. 

I'm sorry I don't really understand what you're asking here? We can wait for @mgorny to come back from holiday and explain better :)

Reply:
the problem is the `ssh` access, which depends on `$HOME/.ssh/authorized_keys`.

*gitolite* wants a git user with home in `/var/lib/gitolite` in order to clone via `git@myhostname.mydomain` -- *gitea* needs `/var/lib/gitea`.

so what about installing them on the same time?
E.g. I would like to run *gitolite* with `gitolite@....` for private stuff and *gitea* with `git@...` for less security relevant stuff on the same server.
Is this still possible with the acct-* packages? What about user who like the other way around?
should this be solved via useflags and depending on them using/creating the git user ?

Reply:
Sorry, this is quite a new thing for us all (I dont know how it should be handled). You could try posting this question into gentoo-dev for reaching common opinion how to solve it. Think there's a ~similar thing going on with gvm acct-*/ ebuilds.

I'm starting to think this could be merged as-is for now, but please start discussion / thinking how to deal with next version bump using GLEP-81 method.

Reply:
@orlitzky would you have an opinion here? Two packages using `git` user with different `/var/lib/${PN}` as home dir?